chapters/functions.tex

```diff
@@ -861,16 +838,12 @@ \subsection{Scalar Functions Applied to Array Arguments}\label{scalar-functions-
   fashion, and the result has the type specified by the function
   definition.
 \item
-  The result of the function call expression is an n-dimensional array
-  with the same dimension sizes as the foreach arguments. Each element
-  ei,..,j is the result of applying f to arguments constructed from the
-  original arguments in the following way:
+  The result of the function call expression is an $n$-dimensional array with the same dimension sizes as the foreach arguments.  Each element \lstinline!ei!, \ldots, \lstinline!j! is the result of applying \lstinline!f! to arguments constructed from the original arguments in the following way:
```

Comment:
This should be cleared up as it didn't make sense before either, I believe it should be 'Each element \lstinline!e[i,$\ldots$,j]! is the ...'

Reply:
Fixed.  The introduction to the list of rules was also rephrased a bit to avoid stray passive sentence.

Please verify fix.